### PR TITLE
Lectures CRU

### PR DESCRIPTION
**Pull request by lesykgg**
Fixed #431

### CODE REVIEW

File: components/courses/app/models/courses/lecture.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    self.table_name = 'courses_lectures'
+
+    extend FriendlyId
+    friendly_id :title, use: :slugged
+
+    belongs_to :season
+    belongs_to :venue
+    belongs_to :mentor
+
+    UPCOMING       = :upcoming
```
Comment (sveredyuk) [inline components/courses/app/models/courses/lecture.rb]: spaces

**Review by n-gb — COMMENTED** [review]
What about screenshots?

**Comment by sveredyuk** [comment]
Screenshots welcomed!

**Comment by lesykgg** [comment]
`#new`
image
`#index`
image
`#edit`
image
@sveredyuk

File: components/courses/app/finders/admin/courses/lecture/lectures_list.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+module Admin
+  module Courses
+    module Lecture
+      class LecturesList < ApplicationFinder
+        def initialize(season)
+          @season = season
+        end
+
+        def call
+          ::Courses::Lecture.includes(mentor: :user).includes(:venue).where(season_id: @season.id)
```
Comment (sveredyuk) [inline components/courses/app/finders/admin/courses/lecture/lectures_list.rb]:
```ruby
::Courses::Lecture
  .includes(mentor: :user)
  .includes(:venue)
  .where(season_id: @season.id)
```

Comment (sveredyuk) [reply components/courses/app/finders/admin/courses/lecture/lectures_list.rb]: good style, imho

File: components/courses/app/helpers/admin/courses/course_helper.rb [hunk 3]
```diff
@@ -9,32 +9,38 @@ def admin_courses_seasons_nav(&block)
...
         link_to icon(:student, t('students.plural')),
-          admin_courses_season_students_path(current_season),
-          class: admin_courses_active_item?('students')
+                admin_courses_season_students_path(current_season),
+                class: admin_courses_active_item?('students')
+      end
+
+      def admin_courses_season_lectures_link
+        link_to icon(:university, t('lectures.plural')),
+                admin_courses_season_lectures_path(current_season),
```
Comment (sveredyuk) [inline components/courses/app/helpers/admin/courses/course_helper.rb:42]: nice.. looks very nice

**Pull request merged**